<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grade_calculator</class>
 <widget class="QMainWindow" name="grade_calculator">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QComboBox" name="choose">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>107</y>
      <width>231</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="hw1">
       <property name="text">
        <string>hw1</string>
       </property>
       <property name="buddy">
        <cstring>slider1</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="slider1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spinbox1"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="hw2">
       <property name="text">
        <string>hw2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="slider2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinbox2"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="hw3">
       <property name="text">
        <string>hw3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="slider3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="spinbox3"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="hw4">
       <property name="text">
        <string>hw4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="slider4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="spinbox4"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="hw5">
       <property name="text">
        <string>hw5</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSlider" name="slider5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QSpinBox" name="spinbox5"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="hw6">
       <property name="text">
        <string>hw6</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSlider" name="slider6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QSpinBox" name="spinbox6"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="hw7">
       <property name="text">
        <string>hw7</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSlider" name="slider7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QSpinBox" name="spinbox7"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="hw8">
       <property name="text">
        <string>hw8</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSlider" name="slider8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QSpinBox" name="spinbox8"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="1" colspan="2">
      <widget class="QSlider" name="slider_f">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QSpinBox" name="spinbox_f"/>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QSlider" name="slider_m2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="spinbox_m2"/>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="spinbox_m1"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="final_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>       final</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="midterm1">
       <property name="text">
        <string>   midterm1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QSlider" name="slider_m1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="midterm2">
       <property name="text">
        <string>   midterm2</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>248</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QRadioButton" name="schemaA">
     <property name="text">
      <string>Schema A</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="schemaB">
     <property name="text">
      <string>Schema B</string>
     </property>
    </widget>
    <widget class="QLabel" name="overall_score">
     <property name="text">
      <string>Overall Score:</string>
     </property>
    </widget>
    <widget class="QLabel" name="final_score">
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="clear">
     <property name="text">
      <string>clear</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
